Sources/Controllers/Resources/OAManageResourcesViewController.mm

```diff
@@ -425,7 +425,7 @@ - (void) willRotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOri
 
 - (BOOL) shouldHideBanner
 {
-    return _currentScope == kLocalResourcesScope || _iapHelper.subscribedToLiveUpdates || (self.region == _app.worldRegion && [_iapHelper isAnyMapPurchased]) || (self.region != _app.worldRegion && [self.region isInPurchasedArea]) || [self.region.regionId isEqualToString:_otherRegionId];
+    return _currentScope == kLocalResourcesScope || _iapHelper.subscribedToLiveUpdates || (self.region == _app.worldRegion && [_iapHelper isAnyMapPurchased]) || (self.region != _app.worldRegion && [self.region isInPurchasedArea]) || [self.region.regionId isEqualToString:_otherRegionId] || [self.region.regionId hasPrefix:OsmAnd::WorldRegions::AntarcticaRegionId.toNSString()];
```

Comment:
Doesn't seem to be right. Maybe it would be better to make the product free and check for that while deciding whether to show the banner or not